src/styles/blocks/caledar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  width: $calendar-widht;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-gap;
+  padding: $calendar-padding;
+
+  @each $number,
+    $day in $days {
+    &--start-day-#{$day} :first-child {
+      margin-left: $number * $calendar-day-space;
+    }
+  }
+
+  @for $day from 28 through 31 {
+    &--month-length--#{$day} {
+      .calendar__day:nth-child(n + #{$day + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &__day {
+    box-sizing: border-box;
+    background-color: $grey;
+    width: $squere-side;
+    height: $squere-side;
+    border: $calendar-day-border solid black;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: 0.5s;
```

Comment:
[Check list](https://github.com/mate-academy/layout_calendar/blob/master/checklist.md) point 8th: Make sure to list all styles that you apply transition to (don't use `all` either)